applications/nrf5340_audio/src/bluetooth/Kconfig

```
@@ -230,29 +230,35 @@ config BLE_ISO_RX_STATS_S
 	int "Interval in seconds to print Bluetooth LE ISO RX stats. 0 to deactivate"
 	default 0
 
-choice BT_AUDIO_UNICAST_BAP_CONFIGURATION
+config CODEC_CAPAB_COUNT_MAX
```

```suggestion
config CODEC_CAP_COUNT_MAX
```

I think CODEC_CAP is used most places

---

I think it was changed because it can be confused with Common Audio Profile. But since it doesn't have the prefix BT_AUDIO it should be fine to change to CAP in this case.

---

Since it says CODEC before, it should be fine to call it CAP